src/index.html

```diff
@@ -1,13 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>DataPortal</title>
-  <base href="/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-</head>
-<body>
-  <app-root></app-root>
-</body>
+  <head>
```

Comment:
Also whitespace will not be compacted in this file contrary to the template files. So we should keep it minimal to keep the file short.

Reply:
This was done by Material's ng add script

Reply:
In that case we can keep the changed formatting.

Just make sure the `window.config` setting is contained because it's expected and modified by the `run.js` script.

Reply:
It looks like in your last merge operation, the formatting was undone and the `body.class` was removed. I suggest you keep the file as you had it first and just add the line with the script for setting the `window.config`.